hello-strongbox-maven/pom.xml

```diff
@@ -17,22 +20,22 @@
 
     <repositories>
         <repository>
-            <id>releases</id>
+            <id>strongbox-releases</id>
             <url>http://localhost:48080/storages/storage0/releases/</url>
         </repository>
         <repository>
-            <id>snapshots</id>
+            <id>strongbox-snapshots</id>
             <url>http://localhost:48080/storages/storage0/snapshot/</url>
```

Comment:
Also, it seems the the repository's `url` has a typo (probably made by me)... It should be:

````
<url>http://localhost:48080/storages/storage0/snapshot/</url>
```